service-front/docker/app/Dockerfile

```diff
@@ -50,9 +50,6 @@ COPY --chown=root:root service-front/data /app/data
 COPY --chown=root:root service-front/module /app/module
 COPY --chown=root:root service-front/public /app/public
 
-# For tests in CI
-COPY --chown=root:root service-front/phpunit.xml /app/phpunit.xml
```

Comment:
ditto on removing the tests directory

Reply:
I have added to the dockerignore, lines to ensure it doesn't get copied over